src/tags/object/Paragraphs/HtxParagraphs.js

```diff
@@ -397,6 +413,41 @@ class HtxParagraphsView extends Component {
         return false;
       });
     });
+
+
+    if (isFF(FF_LSDV_E_278) && this.props.item.contextscroll && item.playingId >= 0 && this.lastPlayingId !== item.playingId && this.state.canScroll) {
+      const _start = this.props.item._value[item.playingId].start;
+      const _end = this.props.item._value[item.playingId].end;
+      const _phaseHeight = root.querySelectorAll('div')[item.playingId]?.offsetHeight || 0;
+      const _duration = this.props.item._value[item.playingId].duration || _end - _start;
+      const _wrapperHeight = root.offsetHeight;
+      const _wrapperOffsetTop = root.querySelectorAll('div')[item.playingId]?.offsetTop - 8; // 8 is the padding between the phrases, so it will keep aligned with the top of the phrase
```

Comment:
can we define the 8 as a const variable some where so we can set it once and reuse it as needed